package.json

```diff
@@ -17,7 +17,10 @@
   "scripts": {
     "start": "grunt",
     "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "postinstall": [
+      "bower install --config.interactive=false",
+      "./node_modules/protractor/bin/webdriver-manager update"
```

Comment:
Something like this in the Grunt file would probably suffice.. 
https://github.com/meanjs/mean/blob/master/gulpfile.js#L198